packages/ag-charts-community/src/chart/axis/axisTitle.ts

```diff
@@ -20,19 +20,19 @@ import { Caption } from '../caption';
 
 export class AxisTitle implements AgAxisCaptionOptions {
     @Validate(BOOLEAN)
-    enabled = false;
+    enabled: boolean = false;
 
     @Validate(STRING, { optional: true })
-    text?: string = undefined;
+    text?: string;
```

Comment:
The JS engine will cope better if you pre-initialize these to undefined so the shape of the object doesn’t change

Reply:
I'm all in favor of testing it.
Personally I don't think it matters because the decorator makes sure the property is declared onto the class prototype, so even if  not initialised the structure never changes.